apps/prairielearn/assets/scripts/lib/tabulator.ts

```diff
@@ -0,0 +1,134 @@
+import { parseHTMLElement } from '@prairielearn/browser-utils';
+import { html } from '@prairielearn/html';
+import { v4 as uuid } from 'uuid';
+import {
+  Tabulator,
+  FormatModule,
+  EditModule,
+  FilterModule,
+  SortModule,
+  PageModule,
+  MutatorModule,
+  FrozenColumnsModule,
+  ResizeTableModule,
+  Options,
+} from 'tabulator-tables';
+
+Tabulator.registerModule([
+  FormatModule, // custom formatters
+  EditModule, // editors, also required for header filters
+  FilterModule, // column header filters
```

Comment:
When I add a filter and then click "x" on the filter to remove it, it is removed from the filter input but the table is still filtered. This is different to the bootstrap-table behavior and I find the bootstrap-table version to be what I would intuitively expect.

Reply:
I just tested and I don't see a difference in behaviour. What you may notice is that the table takes a few milliseconds to update after clicking the "x", which is consistent with the rest of the filter behaviour (the table doesn't update with every stroke to avoid excessive redrawing), and also consistent with what I see in bootstrap-table. Is this what you see?